Can't install eslint

I followed the install instructions: `npm install -g eslint` and then this:

```
$ eslint --init
? How would you like to configure ESLint? Use a popular style guide
? Which style guide do you want to follow? Standard
? What format do you want your config file to be in? JavaScript
Installing eslint-plugin-standard, eslint-config-standard
@mperham/sidekiq@0.5.0 /Users/mike/src/sjs
├── UNMET PEER DEPENDENCY eslint@^2.0.0-rc.0
├── eslint-config-standard@5.1.0 
├── UNMET PEER DEPENDENCY eslint-plugin-promise@^1.0.8
├── eslint-plugin-standard@1.3.2 
└── sidekiq@1.1.1  extraneous

npm WARN eslint-config-standard@5.1.0 requires a peer of eslint@^2.0.0-rc.0 but none was installed.
npm WARN eslint-config-standard@5.1.0 requires a peer of eslint-plugin-promise@^1.0.8 but none was installed.
npm WARN @mperham/sidekiq@0.5.0 No repository field.
```

Running `eslint lib`  gives this:

```
$ eslint lib

Oops! Something went wrong! :(

ESLint couldn't find the plugin "eslint-plugin-standard". This can happen for a couple different reasons:
[snip]
```

I'm new to the JavaScript world - please let me know if you need more info.

```
$ more .eslintrc.js 
module.exports = {
    "extends": "standard",
    "plugins": [
        "standard"
    ]
};

$ npm ls
@mperham/sidekiq@0.5.0 /Users/mike/src/sjs
├── UNMET PEER DEPENDENCY eslint@^2.0.0-rc.0
├── eslint-config-standard@5.1.0
├── UNMET PEER DEPENDENCY eslint-plugin-promise@^1.0.8
├── eslint-plugin-standard@1.3.2
├── generic-pool@2.4.1
├─┬ ioredis@1.15.1 invalid
│ ├── bluebird@2.10.2
│ ├─┬ debug@2.2.0
│ │ └── ms@0.7.1
│ ├── double-ended-queue@2.1.0-0
│ ├── flexbuffer@0.0.6
│ └── lodash@3.10.1
├── minimist@1.2.0 invalid
└── sidekiq@1.1.1 extraneous

npm ERR! peer dep missing: eslint@^2.0.0-rc.0, required by eslint-config-standard@5.1.0
npm ERR! peer dep missing: eslint-plugin-promise@^1.0.8, required by eslint-config-standard@5.1.0
npm ERR! invalid: ioredis@1.15.1 /Users/mike/src/sjs/node_modules/ioredis
npm ERR! invalid: minimist@1.2.0 /Users/mike/src/sjs/node_modules/minimist
npm ERR! extraneous: sidekiq@1.1.1 /Users/mike/src/sjs/node_modules/sidekiq
```

--- a/lib/config/config-initializer.js
+++ b/lib/config/config-initializer.js
@@ -50,6 +50,10 @@ function writeFile(config, format) {
 
     ConfigFile.write(config, "./.eslintrc" + extname);
     log.info("Successfully created .eslintrc" + extname + " file in " + process.cwd());
+
+    if (config.installedESLint) {
+        log.info("ESLint was installed locally. We recommend using this local copy instead of your globally-installed copy.");
+    }
 }
 
 /**
@@ -76,12 +80,24 @@ function installModules(config) {
     if (modules.length === 0) {
         return;
     }
+
+    // Add eslint to list in case user does not have it installed locally
+    modules.unshift("eslint");
+
     installStatus = npmUtil.checkDevDeps(modules);
 
     // Install packages which aren't already installed
     modulesToInstall = Object.keys(installStatus).filter(function(module) {
-        return installStatus[module] === false;
+        var notInstalled = installStatus[module] === false;
+
+        if (module === "eslint" && notInstalled) {
+            log.info("Local ESLint installation not found.");
+            config.installedESLint = true;
+        }
+
+        return notInstalled;
     });
+
     if (modulesToInstall.length > 0) {
         log.info("Installing " + modulesToInstall.join(", "));
         npmUtil.installSyncSaveDev(modulesToInstall);

--- a/tests/lib/config/config-initializer.js
+++ b/tests/lib/config/config-initializer.js
@@ -183,19 +183,19 @@ describe("configInitializer", function() {
             it("should support the google style guide", function() {
                 var config = init.getConfigForStyleGuide("google");
 
-                assert.deepEqual(config, {extends: "google"});
+                assert.deepEqual(config, {extends: "google", installedESLint: true});
             });
 
             it("should support the airbnb style guide", function() {
                 var config = init.getConfigForStyleGuide("airbnb");
 
-                assert.deepEqual(config, {extends: "airbnb", plugins: ["react"]});
+                assert.deepEqual(config, {extends: "airbnb", installedESLint: true, plugins: ["react"]});
             });
 
             it("should support the standard style guide", function() {
                 var config = init.getConfigForStyleGuide("standard");
 
-                assert.deepEqual(config, {extends: "standard", plugins: ["standard"]});
+                assert.deepEqual(config, {extends: "standard", installedESLint: true, plugins: ["standard"]});
             });
 
             it("should throw when encountering an unsupported style guide", function() {
@@ -207,7 +207,13 @@ describe("configInitializer", function() {
             it("should install required sharable config", function() {
                 init.getConfigForStyleGuide("google");
                 assert(npmInstallStub.calledOnce);
-                assert.deepEqual(npmInstallStub.firstCall.args[0], ["eslint-config-google"]);
+                assert.deepEqual(npmInstallStub.firstCall.args[0][1], "eslint-config-google");
+            });
+
+            it("should install ESLint if not installed locally", function() {
+                init.getConfigForStyleGuide("google");
+                assert(npmInstallStub.calledOnce);
+                assert.deepEqual(npmInstallStub.firstCall.args[0][0], "eslint");
             });
         });
 

